src/system-script.js

```diff
@@ -17,6 +17,23 @@ window.define = window.canopyDefine = function(name, deps, m) {
 
 window.define.amd = true;
 
+const ogSystemDelete = outerSystem.delete;
+
+outerSystem.delete = function(normalizedName) {
+  // This is asynchronous, but SystemJS.delete is synchronous. We just accept the race condition :'(
+  SystemJS.import('sofe').then(sofe => sofe.locate({address: normalizedName})).then(url => {
```

Comment:
This is pretty canopy-specific -- wasn't sure if that was a problem or not for this project.